samples/word/99-preview-apis/manage-fields.yaml

```diff
@@ -0,0 +1,120 @@
+id: word-manage-fields
+name: Get fields
+description: This sample shows how to get existing fields.
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#run").click(() => tryCatch(run));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function run() {
+          await Word.run(async (context) => {
+            let field = context.document.body.fields.getFirstOrNullObject();
+            field.load();
+
+            await context.sync();
+
+            console.log("Code: " + field.code);
+            console.log("Result: " + field.result);
+
+            let parentBody = field.parentBody;
+            parentBody.load("text");
+            await context.sync();
+
+            console.log("Text of parentBody: " + JSON.stringify(parentBody.text));
```

Comment:
For a Script Lab perspective, I found printing all three of these to the console to be confusing. This parent text took over the console, making me think it was a field. Consider omitting this log or separating them into different buttons.